public/app/features/explore/RichHistory/RichHistoryQueriesTab.tsx

```diff
@@ -241,11 +269,25 @@ export function RichHistoryQueriesTab(props: RichHistoryQueriesTabProps) {
           })}
         {partialResults ? (
           <div>
-            Showing {queries.length} of {totalQueries} <Button onClick={loadMoreRichHistory}>Load more</Button>
+            <Trans
+              i18nKey="explore.rich-history-queries-tab.showing-queries"
+              defaults="Showing {{ shown }} of {{ total }} <0>Load more</0>"
+              values={{ shown: queries.length, total: totalQueries }}
+              components={[
+                <Button onClick={loadMoreRichHistory} key="loadMoreButton">
+                  Load more
+                </Button>,
```

Comment:
Did you exclude this button from translations on purpose?

Reply:
@L-M-K-B No. I've used the approach mentioned in this GitHub link https://github.com/grafana/grafana/blob/327ae398e6438de7d8b76e55af6ae10d82749104/contribute/internationalization.md#react-components-and-html-tags for translation. I have used default translation template with placeholders for dynamic values ({{ shown }} and {{ total }}), HTML tags (<0>Load more</0>), and components.

Reply:
Ah, I see. Didn't notice the Trans component wrapping the button.